django/core/signing.py

```diff
@@ -8,7 +8,7 @@
 
 There are two components here, separated by a ':'. The first component is a
 URLsafe base64 encoded JSON of the object passed to dumps(). The second
-component is a base64 encoded hmac/SHA1 hash of "$first_component:$secret"
+component is a base64 encoded hmac/SHA-256 hash of "$first_component:$secret"
```

Comment:
We can also update an exception raised by `signing.loads()` to:
```
BadSignature: Signature "9m1V63yIglMNHcSDjNRZSb02ZYG9fP_QjvflQHBajf4-modified" does not match
```

Reply:
I'm sorry but I don't follow? The patch only changes a docstring, no code. It shouldn't change any hashes in the testsuite?

Reply:
I'm talking about an example in the same docstring:
https://github.com/django/django/blob/1d071ec1aa8fa414bb96b41f7be8a1bd01079815/django/core/signing.py#L20

Reply:
Good catch, thanks! I haven't noticed the change to the exception message.

The full exception message reads `django.core.signing.BadSignature: Signature "...-modified" does not match` here, but I think that the module path isn't particularly helpful in the docstring so I didn't add it.

Reply:
I had to shorten the signature to fix the line length error. I think the example doesn't lose clarity.